src/game/cGangs.cpp

```diff
@@ -390,7 +390,7 @@ sGang* cGangManager::GetTempWeakGang()
     for (int & m_Skill : newGang->m_Skills)    m_Skill = g_Dice % 30 + 51;
     for (int & m_Stat : newGang->m_Stats)    m_Stat = g_Dice % 30 + 51;
     newGang->m_Stats[STAT_HEALTH] = 100;
-    newGang->set_weapon_level( g_Dice.in_range(0, 1) );
```

Comment:
A complex way of saying `set_weapon_level(0);` -- so I think it's supposed to be 0 or 1.

Reply:
I guess so